src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/resource_allocation_manager_impl.cc

```diff
@@ -205,14 +205,14 @@ EnumType StringToEnum(const std::string& str) {
 void ConstructOnRCStatusNotificationParams(
     smart_objects::SmartObject& msg_params,
     const std::map<std::string, uint32_t>& allocated_resources,
-    const std::vector<std::string>& supported_resources) {
+    const std::vector<std::string>& supported_resources,
+    const uint32_t app_id) {
```

Comment:
@LitvinenkoIra Is this mobile app_id or hmi_app_id? Please specify.

Reply:
@LitvinenkoIra @AByzhynar in such dunctions it would be better to use internal app id. SDL is able automaticaly convert app_id to hmi/mobile during sending RPC to mobile ot HMI as I remember.